src/main/java/com/faforever/client/game/PlayerCardController.java

```diff
@@ -213,6 +216,12 @@ private Integer getRatingChange(LeaderboardRatingJournalBean ratingJournal) {
     return null;
   }
 
+  public void removeAvatar() {
+    this.avatarImageView.managedProperty().set(false);
+    this.avatarStackPane.managedProperty().set(false);
+    this.avatarStackPane.visibleProperty().set(false);
```

Comment:
You set the avatar to invisible here.

Also just use the setVisible method rather than going through the property.

And in the initialize use the utility function JavafxUtil.bindManagedToVisible on the stack pane.

Reply:
Fair enough, I meant that without any changes I havent seen an avatar anyway (like on the 12.3 client)